tst/EnergyPlus/unit/Fixtures/HVACFixture.hh

```diff
@@ -6,6 +6,24 @@
 
 // EnergyPlus Headers
 #include "EnergyPlusFixture.hh"
+#include <EnergyPlus/CurveManager.hh>
```

Comment:
Note that clear_state tear downs are not the only reason for many of the includes; sometimes the unit tests are accessing routines and data.  I will make this change but it seems to me to make HVACFixture less useful because then the includes need to be repeated in different unit tests.